### ISSUE TITLE
Can't see any SMTP logs

### ISSUE THREAD

**Issue by iloco** [StamusNetworks/SELKS#11]
Hello,
I'm a new user using SELKS in production and I'm just discovering Kibana dashboads.
I can see other flow but I'm surprised not to see any SMTP logs in this dashboard:
https://myselkserver/log/#/dashboard/elasticsearch/SMTP

It's a fresh install and this tuto has already been followed and aplicated:
https://github.com/StamusNetworks/SELKS/wiki/Running-SELKS-in-production

I surrely missed something, but what ?

**Comment by pevma** [comment]
Do you have "event_type":"smtp" logs in /var/log/suricata/eve.json ?

**Comment by iloco** [comment]
Yes, I have some SMTP logs in that file:
(root@SELKS:/var/log/suricata# tail -f eve.json | grep smtp)

But still nothing in my dashboard...should I configure something to see them ?

**Comment by pevma** [comment]
Just to confirm , can you try:
grep '"event_type":"smtp"' /var/log/suricata/eve.json  |more

The should be displaying by default.
Also - are those logs recent - the dashboard shows by default - the last 24 hrs?

**Comment by iloco** [comment]
Yes it's working, I can se them right now but I had to reset dashboards to default ones:
rm /etc/kibana/dashboards-loaded
/etc/init.d/kibana-dashboards-stamus start

Yesterday evening I tried to Update dashboards following this link:
https://github.com/StamusNetworks/SELKS/wiki/How-to-load-or-update-dashboards

Now SMTP-total increase normaly but PDF-Attachments (0) Word-Attachments (0), curious no ?
For exemple:
root@SELKS:~# grep '"event_type":"smtp"' /var/log/suricata/eve.json |more
{"timestamp":"2015-04-06T18:04:06.605691","flow_id":24013792,"in_iface":"eth1","event_type":"smtp","src_ip":"myip","src_port":49260,"dest_ip":"myserver","dest_port":25,"proto":"TCP","smtp":{"from":"Olive <me@mydomain.fr>","to":["dest@domain.fr"],"subject":"test de mess avec PJ","attachment":["SFR.doc"]}}
{"timestamp":"2015-04-06T18:42:49.954549","flow_id":32298064,"in_iface":"eth1","event_type":"smtp","src_ip":"myip","src_port":49473,"dest_ip":"myserver","dest_port":25,"proto":"TCP","smtp":{"from":"Olive <me@mydomain.fr>","to":["dest@domain.fr"],"subject":"Fwd: Facture_20613669_pdf.pdf - Adobe Acrobat Professional","attachment":["Facture_20613669_pdf.pdf"]}}

**Comment by pevma** [comment]
What is the output of:
dpkg -l |grep kibana-dashboards
? thanks

**Comment by pevma** [comment]
Ok that is the latest version of the dashboards.

In that case  - can you try seeing if it is in the ES at all. You can just start a "default" dashboard and search for the file name - Facture_20613669_pdf - would it return any results ? (or narrow down the time line search to display only that minute)
